code/modules/projectiles/projectile/magic.dm

```diff
@@ -158,7 +158,8 @@
 						if("drone")
 							new_mob = new /mob/living/simple_animal/drone(M.loc)
 							var/mob/living/simple_animal/drone/D = new_mob
-							D.update_drone_hack()
+							spawn(50) //this should give us time for the rest of the proc to finish so a mob will be in the drone
```

Comment:
I'd have to add an istype check for the drone again way down at the bottom and that'd also be gross

I could also remove the !mind restrction on the drone hack but then they'd miss their law update.

All solutions are shit